src/com.frostycoolslug.goxlr-utility.sdPlugin/app/display-volume.js

```diff
@@ -58,19 +55,21 @@ class VolumeMonitor {
 
     #event_handle = () => {};
 
-    constructor(context, serial, mix, channel_a, channel_b) {
+    constructor(context, serial, mix, channel_a, channel_b, hide_name) {
         this.context = context;
         this.serial = serial;
         this.mix = mix;
         this.channel_a = channel_a;
         this.channel_b = channel_b;
-
+        this.hide_name = hide_name;
         this.monitor_a = `/mixers/${serial}/levels/volumes/${channel_a}`;
         this.monitor_b = `/mixers/${serial}/levels/submix/inputs/${channel_b}/volume`;
 
         // Monitor for Submix Enable / Disable...
         this.monitor_submix = `/mixers/${serial}/levels/submix`;
-        this.device = `/mixers/${serial}`
+        this.device = `/mixers/${channel_b}`
+
+        this.fader_status = [ `/mixers/${serial}/fader_status/A/mute_state`, `/mixers/${serial}/fader_status/B/mute_state`, `/mixers/${serial}/fader_status/C/mute_state`, `/mixers/${serial}/fader_status/D/mute_state`];
```

Comment:
I'm wondering if this might be easier to catch any path that starts with `/mixers/${serial}/fader_status/`, that way if the user (for example) changes the channel on a fader in the UI, or even the mute type, the patch event will be picked up here and the button re-rendered to reflect the new state.

Reply:
I can try, though I did try and it didn't seem to work, but I'll give this another shot

Reply:
after testing again, sadly the fader_status object doesn't get an event when one of the faders mute_state chanes

Reply:
What you'll need is:
```
this.fader_status = `/mixers/${serial}/fader_status/`
```

Then in `#onEvent`:
```
|| patch.path.startWith(this.fader_status)
```

Which should trigger an update whenever anything in the fader status path changes

Reply:
I should have thought of using a startswith... I was still thinking of them as event objects instead of them just being strings